src/main/kotlin/net/atos/zac/app/zaken/converter/RESTResultaattypeConverter.kt

```diff
@@ -13,6 +13,7 @@ import net.atos.zac.util.PeriodUtil
 import net.atos.zac.util.UriUtil
 import java.net.URI
 import java.time.Period
+import java.util.*
```

Comment:
This should probably be regular import(s) instead of wildcard

Reply:
Fixed! So annoying. Spotless nor Detekt picks this up.. Should be able to..